README.md

```diff
@@ -1,2 +1,136 @@
+	- **Sass** for styling
+	- **Webpack** for bundling	
+- __Stretch Add-ons__
+	
+	Some stretch and polish goals will use additional technologies.
+	
+	- **AWS S3** for storing product images
+	- **mailgun** or **sendgrid** for sending email
+	
+Your team will be building a minimal CMS to allow users to list personal items for sale.
+The application will allow unauthorized users to browse the listed products. To post an Item for sale, a User account will be required. The backend should provide Restful api endpoints. The frontend should be a SPA that provides a smooth user experience.
+
+You will need to work as a team to complete this project. To facilitate team communication you will be using [Trello](https://trello.com/) to track tasks and stay on track.
+
+
+## Business Requirements
+
+### MVP
+
+#### Unauthenticated user
+- Home page should list top viewed `Items` by `Category`.
+- Home page should provide a way to view **all** `Items`.
+- Home page should provide a view of `Items` by `Category`.
+- Home page should show a subset of key details of the `Items`, along with one image.
+- Home page should have a login button.
+- `Item` detail page should contain:
+	- description (required)
+	- price
+	- manufacturer or make
+	- model name or number
+	- condition (required)
+	- category (required)
+	- dimensions
+	- when the `Item` was initially published
+	- when the `Item` was last updated
+	- notes / details
+
+#### Authenticated User
+- Should be able to view the same home page and details that an unauthenticated user can.
+- `User` home page should contain: 
+	- A way to add new `Items`.
+	- A way to logout.
+	- display the current `Users` name
+	- List of `Items` currently published for sale.
+	- List of `Items` pending publishing.
+		- Should provide a quick way to publish the `Item`.
+	- List of `Items` in other state, (sold, archived)
+- Add new `Item` should present the `User` with a form to add all `Item` details.
+	- A newly added `Item` should have a status of _Unpublished_ when created.
+	- The form should provide validation and feedback if inputs in omitted or invalid.
+	- Should be able to select an image and upload it for the `Item`.
+- `Item` detail owned by `User`:
+	- Should be able to modify all inputs.
+	- Change the _Status_ of the `Item`.
+	- Should be able to replace the image associated with the `Item`.
+- When an `Item` is published it should record the **publish** date and time.
+- When an `Item` is modified it should record the **update** date and time.
+- The `User` **Settings** view should provide the `User` a way to change their password.
+	 
+#### Features
+- The application should display a logo.
+- Mobile friendly application. Styles should work from phone up to desktop.
+- Clicking on an `Item` in any list, should display the detail view of that `Item`.
+- When `Item` details are viewed by anyone other than the owner, it should count the view, to track what `Items` are most popular.
```

Comment:
Might want to also make views a stretch goal. Great idea though but might get in the weeds about how to track uniques vs page refreshes unless it doesn't matter?